service/controller/app/resource/releasemigration/resource.go

```diff
@@ -157,7 +162,7 @@ func (r *Resource) ensureReleasesMigrated(ctx context.Context, k8sClient k8sclie
 		r.logger.LogCtx(ctx, "level", "debug", "message", "migration not complete")
 	}
 
-	b := backoff.NewConstant(5*time.Minute, 10*time.Second)
+	b := backoff.NewConstant(20*time.Minute, 10*time.Second)
```

Comment:
I think 5 mins is too low when checking the progress. Best to just give it more time.